packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1130,23 +1130,24 @@ The performance optimization will only be applied when all three parameters are
 
 The first three parameters in the table below correspond to different types of lists:
 
-- [List All Users](#list-all-users) (no parameters)
-- [Find Users](#find-users) (`q`)
-- [List Users with a Filter](#list-users-with-a-filter) (`filter`)
 - [List Users with Search](#list-users-with-search) (`search`)
+- [List Users with a Filter](#list-users-with-a-filter) (`filter`)
+- [Find Users](#find-users) (`q`)
+- [List All Users](#list-all-users) (no parameters)
 
 | Parameter   | Description                                                                                                                                    | Param Type   | DataType   | Required |
 | :---------- | :--------------------------------------------------------------------------------------------------------------------------------------------- | :----------- | :--------- | :------- |
-| q           | Finds a user that matches `firstName`, `lastName`, and `email` properties                                                                      | Query        | String     | FALSE    |
-| filter      | [Filters](/docs/reference/core-okta-api/#filter) users with a supported expression for a subset of properties                  | Query        | String     | FALSE    |
-| search      | Searches for users with a supported [filtering](/docs/reference/core-okta-api/#filter) expression for most properties          | Query        | String     | FALSE    |
+| search      | Searches for users with a supported [filtering](/docs/reference/core-okta-api/#filter) expression for most properties. Prefer this option for optimal performance         | Query        | String     | FALSE    |
 | limit       | Specifies the number of results returned (maximum 200)                                                                                         | Query        | Number     | FALSE    |
 | after       | Specifies the pagination cursor for the next page of users                                                                                     | Query        | String     | FALSE    |
+| filter      | [Filters](/docs/reference/core-okta-api/#filter) users with a supported expression for a subset of properties                  | Query        | String     | FALSE    |
 | sortBy      | Specifies field to sort by (for search queries only)                                                                                           | Search query | String     | FALSE    |
 | sortOrder   | Specifies sort order asc or desc (for search queries only)                                                                                     | Search query | String     | FALSE    |
+| q           | Finds a user that matches `firstName`, `lastName`, and `email` properties (but prefer search; see above)                       | Query        | String     | FALSE    |
 
   * If you don't specify a value for `limit`, the maximum (200) is used as a default.  If you are using a `q` parameter, the default limit is 10.
```

Comment:
The ordering of this table is weird now.  Besides breaking the earlier sentence about the first three parameters relating to different types of list (more on that in a sec), it weaves those three parameters in and out of the rest.  We lead with our big promoted `search` type, then mention a couple optional parameters that apply to all three types, then we admit to a second type, then a couple more parameters that apply _only to the search case_, then finally poor `q` whose smaller default value for `limit` is tucked away in a note even further down.

Just keep the three types as the first three parameters, then list the modifiers.  Sure, put `search` first, but put the three as the first three.  That way the earlier sentence can continue to talk about the first three parameters in the table.  But speaking of that, it calls them "three types of list", which is specious.  I challenge you to come up with three adjectives to describe those three types.  Those three parameters "correspond to different ways to specify which users to include in the list".

Reply:
Alright, fine.  And you're right, "different types of lists" is pure nonsense, I'll fix it.